tests/foreman/cli/test_discoveredhost.py

```diff
@@ -30,11 +30,10 @@
 from robottelo.datafactory import gen_string
 from robottelo.decorators import skip_if_not_set
 from robottelo.libvirt_discovery import LibvirtGuest
-from robottelo.test import CLITestCase
 
 
 @pytest.mark.run_in_one_thread
-class DiscoveredTestCase(CLITestCase):
+class TestDiscoveredHost:
```

Comment:
is there a need for this to be a class instead of module-level tests?

Reply:
There is not, but if I used module-level tests I would need to mark every single test to run in one thread. Personally I prefer marking whole class, but I can change that.

Reply:
You can mark the module, which will apply the mark to all test cases.

```pytestmark=[pytest.mark.run_in_one_thread]```  At the top of the module.

Reply:
Thanks, didn't know about this option.

Reply:
Yep! It's not a perfect solution, if there is any test case where you _don't_ want that mark, you're SOL, there's no clean way to remove the mark on that single test.